cmd/root.go

```diff
@@ -32,6 +33,8 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.console-for-sap.yaml)")
 	rootCmd.AddCommand(webapp.NewWebappCmd())
+	rootCmd.AddCommand(agent.NewRegisterCmd())
+	rootCmd.AddCommand(agent.NewStartCmd())
```

Comment:
This doesn't look quite right. The `start` and `register` commands should be children sub-commands of an intermediate `agent` command, which will be child of the root one.

Reply:
@stefanotorresi please, look again.